crates/runtime/src/dataconnector/iceberg.rs

```diff
@@ -79,7 +85,12 @@ impl DataConnectorFactory for IcebergDataConnectorFactory {
     }
 
     fn parameters(&self) -> &'static [ParameterSpec] {
-        crate::catalogconnector::iceberg::PARAMETERS
+        // Concat the data connector specific parameters with the catalog parameters
+        let iceberg_params = PARAMETERS.to_vec();
+        let catalog_params = crate::catalogconnector::iceberg::PARAMETERS.to_vec();
+        let mut params = iceberg_params;
+        params.extend(catalog_params);
+        Box::leak(Box::new(params)) // Leak the memory to return a static slice
```

Comment:
The whole point of this being a static slice is that its known at compile time - surely there is a way to have it return a static slice here? Maybe with something similar to what was done for models? https://github.com/spiceai/spiceai/blob/trunk/crates/runtime/src/model/params/mod.rs#L47